src/SortingAlgorithms.jl

```diff
@@ -61,7 +61,14 @@ uint_mapping(o::Lt,   x     ) = error("uint_mapping does not work with general L
 const RADIX_SIZE = 11
 const RADIX_MASK = 0x7FF
 
-function sort!(vs::AbstractVector, lo::Int, hi::Int, ::RadixSortAlg, o::Ordering, ts=similar(vs))
+function sort!(vs::AbstractVector, lo::Int, hi::Int, ::RadixSortAlg, o::Ordering, ts=similar(vs, 0))
+    # Fallback on small lists
```

Comment:
```suggestion
    # Fallback to default algorithm for short vectors as radix sort is slower for them
    # The threshold has been chosen because radix sort allocates an array of that size
    # and validated by benchmarks
```